buttons/imagefile.go

```diff
@@ -91,9 +91,9 @@ func (btn *ImageFileButton) Pressed() {
 }
 
 // NewImageFileButton creates a new ImageFileButton with the specified image on it
-func NewImageFileButton(filePath string) (*ImageFileButton, error) {
+func NewImageFileButton(filePath string, btnSize int) (*ImageFileButton, error) {
```

Comment:
I'd like to make this not need the btnSize passing into it, as the end user shouldn't need to care about it.  I'll open a separate issue about it though.